CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@ The main GitHub repository is at [https://github.com/nhibernate/nhibernate-core]
 
 ## The Build Menu
 
-**ShowBuildMenu.bat** will be your friend throughout this journey. He's easiest to work with if you make his box bigger by following these steps: 
+**ShowBuildMenu.bat** will be your friend throughout this journey. A .sh version exists for Linux. He's easiest to work with if you make his box bigger by following these steps: 
```

Comment:
Why `ShowBuildMenu.bat` is referenced as "He"? Shouldn't it be "It"?

Reply:
Yes.